lib/tlog/play.c

```diff
@@ -95,6 +96,11 @@ tlog_play_create_es_json_reader(struct tlog_errs **perrs,
     }
     baseurl = json_object_get_string(obj);
 
+    /* Check for the verbose flag */
+    if (json_object_object_get_ex(conf, "verbose", &obj)) {
+        verbose = json_object_get_boolean(obj);
+    }
+
```

Comment:
This should rather be something like this:
```c
    verbose = json_object_object_get_ex(conf, "verbose", &obj) &&
              json_object_get_boolean(obj);
```
Otherwise `verbose` could end up uninitialized, if the option is missing.